helm/cluster/templates/_test_registry_helpers.tpl

```diff
@@ -0,0 +1,21 @@
+{{/* Test helpers for registry configuration */}}
+{{- define "cluster.internal.test.awsContainerImageRegistry" -}}
```

Comment:
Where is this template used? I'm not seeing it in here.

Reply:
It's defined in helm/cluster/ci/test-registry-mirrors-values.yaml

Reply:
Oh this is test specific? 🤦

Reply:
Yes 👍🏻